<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWidget</class>
 <widget class="QWidget" name="MenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Argus</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	outline: none;
}
/* QWidget */
QWidget{
	background-color: #2c3e50;
}
/* QFrame */
QFrame#navbar{
	background-color: #2c3e50;
}
/* QLineEdit */
QLineEdit{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	color: #ecf0f1;
	background-color: #2c3e50;
	border: 0px;
	border-bottom: 1px solid #ffffff;
	selection-background-color: #34495e;
	selection-color: #ecf0f1;
}
/* QPushButton */
QPushButton{
	border: 0;
	border-radius: 3px;
	background-color: #8e44ad;
	color: #ecf0f1;
	font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	text-align: center;
}
QPushButton:hover{ background-color: #9b59b6; }
/* MenuWidget */
QWidget#MenuWidget{
	outline: none;
	background-color: #2c3e50;
	color: #ecf0f1;
}
QFrame#navbar QRadioButton{
	padding-right: 9px;
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	text-align: right;
	color: #ecf0f1;
	border: 0;
}
QFrame#navbar QRadioButton:checked  {
	border: 0px;
    border-left: 5px solid #ffffff;
    border-color: #8e44ad;
	background-color: #34495e;
}
QFrame#navbar QRadioButton:hover{ background-color: #34495e; }
QFrame#navbar QRadioButton:indicator { width: 1px; height: 1px; background-color: #2c3e50 }
QFrame#navbar QRadioButton::indicator:hover { background-color: #34495e; }
QFrame#navbar QRadioButton::indicator:checked { background-color: #34495e; }

QWidget#MenuWidget QFrame#header QFrame {
	background-color: #34495e;
}
QWidget#MenuWidget QLineEdit#txtSearch{
	height: 32px;
	font-family: Roboto;
	font-size: 16px;
	color: #ecf0f1;
	background-color: #34495e;
	border: 0px;
	border-bottom: 1px solid #ffffff;
	selection-background-color: #2c3e50;
	selection-color: #ecf0f1;
}
QWidget#MenuWidget QPushButton#btnProfile{
	font-family: Roboto;
	font-size: 16px;
	font-weight: 400;
	color: #ecf0f1;
	border: 0;
	padding: 20px;
	text-align: center;
	background-color: #2c3e50;
}
QWidget#MenuWidget QPushButton#btnProfile:hover{ background-color: #34495e; }
QWidget#MenuWidget QPushButton#btnLogout{
	background-color: #2c3e50;
	border: 0;
}
QWidget#MenuWidget QPushButton#btnLogout:hover{ background-color: #34495e; }
QWidget#MenuWidget QFrame#content{
	color: #000000;
	background-color: #ffffff;
}
QWidget#MenuWidget QLabel{
	font-family: Roboto;
	font-weight: 100;
	font-size: 28px;
	color: #ecf0f1;
}
/* QStackedWidget */
QStackedWidget#content QWidget{
	background-color: #ffffff;
	color: #2c3e50;
}
/* QMessageBox */
QMessageBox{
	background-color: #2c3e50;
}
QMessageBox QLabel{
	background-color: #2c3e50;
	color: #ecf0f1;
	font-family: Roboto;
	font-size: 14px;
}
QMessageBox QPushButton{
	border: 0;
	border-radius: 3px;
	background-color: #8e44ad;
	color: #ecf0f1;
    font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	min-height: 30px;
	min-width: 120px;
}
QMessageBox QPushButton:hover{
	background-color: #9b59b6;
}
QMessageBox QPushButton#btnInformative{
	border: 0;
	border-radius: 3px;
	background-color: #2980b9;
	color: #ecf0f1;
    font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	min-height: 30px;
	min-width: 120px;
}
QMessageBox QPushButton#btnInformative:hover{
	background-color: #3498db;
}
QMessageBox QPushButton#btnWarning{
	border: 0;
	border-radius: 3px;
	background-color: #c0392b;
	color: #ecf0f1;
    font-family: Roboto;
	font-size: 16px;
	font-weight: 300;
	min-height: 30px;
	min-width: 120px;
}
QMessageBox QPushButton#btnWarning :hover{
	background-color: #e74c3c;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="main">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="header">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>85</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frmTitle">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="lblIcon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>0</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap>../resources/logos/light.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Argus</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frmSearch">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../resources/icons/search.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtSearch">
             <property name="placeholderText">
              <string>Pesquisar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frmUser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnProfile">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Júlio César</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../resources/icons/avatar.png</normaloff>../resources/icons/avatar.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLogout">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>85</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../resources/icons/logout.png</normaloff>../resources/icons/logout.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="body">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="navbar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="btnDashboard">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Principal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btnCustomers">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Clientes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btnWorkers">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Funcionários</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btnReports">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Relatórios</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="btnAbout">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Sobre</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="content">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="contentPage1"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
